price-feeder/oracle/provider/huobi.go

```diff
@@ -274,10 +276,12 @@ func (p *HuobiProvider) pong(bz []byte) {
 		return
 	}
 
-	if err := p.wsc.SendJSON(struct {
-		Pong uint64 `json:"pong"`
-	}{Pong: heartbeat.Ping}); err != nil {
-		p.logger.Err(err).Msg("could not send pong message back")
+	for _, conn := range p.wsc.connections {
```

Comment:
This could potentially send a bunch of duplicate pong messages. Maybe we should pass in a pong_handler function to the websocket controller?

Reply:
Or pass the WebsocketConnection object back as one of the `handle_message` args..?